packages/esbuild-plugin/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@vanilla-extract/esbuild-plugin",
+  "version": "0.0.1",
+  "description": "Zero-runtime Stylesheets-in-TypeScript",
+  "main": "dist/vanilla-extract-esbuild-plugin.cjs.js",
+  "module": "dist/vanilla-extract-esbuild-plugin.esm.js",
+  "files": [
+    "/dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/seek-oss/vanilla-extract.git",
+    "directory": "packages/esbuild-plugin"
+  },
+  "author": "SEEK",
+  "license": "MIT",
+  "peerDependencies": {
+    "esbuild": "^0.11.1"
```

Comment:
IMO restrictive peer dependency version ranges mostly just get in the way when dealing with third party libs. Thoughts on making this `>=0.11.1`? This way, if v0.12 comes out, the version range will still match. I realise that we're using experimental esbuild features, but we'll know pretty quickly if this doesn't work in practice, and we'll want to keep up to date anyway.